tests/coverage/async.rs

```diff
@@ -86,6 +86,14 @@ fn l(x: u8) {
 
 async fn m(x: u8) -> u8 { x - 1 }
 
+async fn n() -> u8 { 1 }
```

Comment:
Since this function isn't important on its own, you can put `#[coverage(off)]` on it to help keep the `.cov-map` snapshot simpler.